terraform-common.mk

```diff
@@ -47,4 +47,4 @@ graph:
 	terraform graph -draw-cycles | dot -Tpng > graph.png
 
 $(ENV_NAME).tfvars:
-	$(TOP)/bin/generate-tfvars >$@
+	$(TOP)/bin/generate-tfvars $@
```

Comment:
@henrikhodne this var is resolved by `make` rather than `bash`, though.  It's `$(ENV_NAME).tfvars`.